packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -117,18 +117,65 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
     );
   },
 
+  // The ButtonBar widget in package:flutter/material.dart is planned to be deprecated
+  // in favor of the OverflowBar widget. This ButtonBar implementation uses the
+  // OverflowBar widget internally for backward compatibility. Note that the ButtonBar
```

Comment:
"note that" -> https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose